.github/ISSUE_TEMPLATE/request_identifier.yaml

```diff
@@ -0,0 +1,22 @@
+name: Request Identifier
+description: File a request to add a new identifier type to SocialConnect
+title: "[Identifier Request]: "
+body:
+    - type: input
+      id: identifier
+      attributes:
+          label: Identifier Name
+          description: "Which identifier would you like to add?"
+          placeholder: "Example: Twitter, Discord, Instagram, Telegram, etc..."
+      validations:
+          required: true
+    - type: input
+      id: prefix
+      attributes:
+          label: Preferred Prefix
+          description: "What would be a preferred prefix for the same?"
+          placeholder: "Example: twit, discord, github, phone, mail, tel, etc..."
+    - type: markdown
+      attributes:
+          value: |
+              In order to add support for the requested identifier, you can add support for the identifier [here](https://github.com/celo-org/celo-monorepo/blob/75c223a1b5296ac0fecdb54fb29c52432cf3fece/packages/sdk/identity/src/odis/identifier.ts#L44) and open a PR.
```

Comment:
this was super recently updated and moved to the base package (part of ASv1 sunsetting) as part of [this PR](https://github.com/celo-org/celo-monorepo/pull/10165/files#diff-e4cbdb5c546d4fb22beb046136269ebf436202e9c0a6262c6e1bdc0ab3c16125), could you update the link? 🙏

Reply:
Done!